src/internal/easee/controller.go

```diff
@@ -97,3 +113,64 @@ func (c *controller) ElectricityMeterReport(unit string) (float64, error) {
 		return 0, errors.Errorf("unsupported unit: %s", unit)
 	}
 }
+
+func (c *controller) SetChargepointChargingMode(mode string) error {
+	switch mode {
+	case ChargingModeSlow:
+		return c.client.SetChargingCurrent(c.chargerID, SlowChargingCurrentInAmpers)
```

Comment:
Isn't it like you cannot set the charging mode if the session is not started?

Reply:
Outdated.